src/CommonLib/hostfxr_utility.h

```diff
@@ -51,8 +51,28 @@ class HOSTFXR_UTILITY
 
     static
     HRESULT
-    FindDotnetExePath(
+    GetAbsolutePathToDotnet(
         STRU*   struDotnetLocation
     );
+
+    static
+    HRESULT
+    GetAbsolutePathToHostFxr(
+        _In_ STRU* struAbsolutePathToDotnet,
```

Comment:
coding guideline on parameter naming. pointer should start with p***
similar to other functions